frontend/components/ReusableComponents/PageHeader.vue

```diff
@@ -0,0 +1,49 @@
+<script setup>
+import PulseBackgroundAnimation from '@/components/ReusableComponents/PulseBackgroundAnimation.vue'
+
+defineProps({
+  // meetup: {
+  //   type: Object,
+  //   default: () => {},
+  // },
+  text1: {
```

Comment:
I think I will rename this text1, text2, text3 to something little more descriptive as: title, firstSubtitle, secondSubtitle.

Reply:
These "text1", "text2" and "text3" are single words, that's why the middle word can be red, and 2 others black.
I can change it for "word1" if it will be less annoying :P

Reply:
Sorry, I didn't get it ;p. So please change this.